examples/rl/sb3_simple.py

```diff
@@ -0,0 +1,165 @@
+""" Optuna example that optimizes the hyperparameters of
+a reinforcement learning agent using A2C implementation from Stable-Baselines3
+on a OpenAI Gym environment.
+
+This is a simplified version of what can be found in https://github.com/DLR-RM/rl-baselines3-zoo.
+
+You can run this example as follows:
+    $ python sb3_simple.py
+
+"""
+from typing import Any
+from typing import Dict
+
+import gym
+from stable_baselines3 import A2C
+from stable_baselines3.common.callbacks import EvalCallback
+import torch as th
+import torch.nn as nn
+
+import optuna
+from optuna.pruners import MedianPruner
+from optuna.samplers import TPESampler
+
+
+N_TRIALS = 100
+N_JOBS = 2
+N_STARTUP_TRIALS = 5
+N_EVALUATIONS = 2
+N_TIMESTEPS = int(2e4)
+EVAL_FREQ = int(N_TIMESTEPS / N_EVALUATIONS)
+N_EVAL_EPISODES = 3
+
+ENV_ID = "CartPole-v1"
+
+DEFAULT_HYPERPARAMS = dict(policy="MlpPolicy", env=ENV_ID)
+
+
+def sample_a2c_params(trial: optuna.Trial) -> Dict[str, Any]:
+    """Sampler for A2C hyperparameters.
+
+    :param trial: (optuna.Trial)
+    :return: (Dict[str, Any])
+    """
+    gamma = trial.suggest_categorical("gamma", [0.9, 0.95, 0.98, 0.99, 0.999, 0.9999])
+    max_grad_norm = trial.suggest_categorical("max_grad_norm", [0.3, 0.5, 0.7, 0.9, 1, 2, 5])
+    gae_lambda = trial.suggest_categorical("gae_lambda", [0.8, 0.9, 0.92, 0.95, 0.98, 0.99, 1.0])
+    n_steps = trial.suggest_categorical("n_steps", [8, 16, 32, 64, 128, 256, 512, 1024])
+    learning_rate = trial.suggest_loguniform("lr", 1e-5, 1)
+    ent_coef = trial.suggest_loguniform("ent_coef", 0.00000001, 0.1)
+    ortho_init = trial.suggest_categorical("ortho_init", [False, True])
+    net_arch = trial.suggest_categorical("net_arch", ["tiny", "small"])
+    activation_fn = trial.suggest_categorical("activation_fn", ["tanh", "relu"])
+
+    net_arch = {"tiny": [dict(pi=[64], vf=[64])], "small": [dict(pi=[64, 64], vf=[64, 64])],}[
+        net_arch
+    ]
+
+    activation_fn = {"tanh": nn.Tanh, "relu": nn.ReLU,}[activation_fn]
+
+    return {
+        "n_steps": n_steps,
+        "gamma": gamma,
+        "gae_lambda": gae_lambda,
+        "learning_rate": learning_rate,
+        "ent_coef": ent_coef,
+        "max_grad_norm": max_grad_norm,
+        "policy_kwargs": dict(
+            net_arch=net_arch, activation_fn=activation_fn, ortho_init=ortho_init
+        ),
```

Comment:
Same here. I would use dict literal.
```suggestion
        "policy_kwargs": {
            "net_arch": net_arch,
            "activation_fn": activation_fn,
            "ortho_init": ortho_init,
        },
```